touch-debug/main/log.script

```diff
@@ -1,4 +1,13 @@
-function init(self) msg.post(".", "acquire_input_focus") end
+local function on_jstodef_input(self, message_id, message)
+    local pressed = message_id == "pointerdown" and "true" or "false"
+    local released = message_id == "pointerup" and "true" or "false"
+    html5.run("onJsToDefInput(" .. pressed .. ", " .. released .. ", " .. message ..")")
+end
+
+function init(self)
+    msg.post(".", "acquire_input_focus")
+    jstodef.add_listener(on_jstodef_input)
+end
```

Comment:
This is the main modification responsible for this observation. Pathway:
- `addEventListener(event, (e) => JsToDef.send(event, e.timeStamp))` sends the event to Defold
- `jstodef.add_listener(on_jstodef_input)` receives the event in Defold
- `html5.run("onJsToDefInput(...)")` sends a signal back to JS within the same frame